### ISSUE TITLE
Does not build on SUSE 11SP4

### ISSUE THREAD

**Issue by jimflood** [sparklemotion/sqlite3-ruby#182]
Unfortunately, it is my recent merged pull request causing this:

gmake[1]: Entering directory `/home/sprotsman/workspace/hinode/vendor/bundle/ruby/2.2.0/bundler/gems/sqlite3-ruby-b161d6587b1c/tmp/x86_64-linux/sqlite3_native/2.2.4'
compiling ../../../../ext/sqlite3/statement.c
compiling ../../../../ext/sqlite3/database.c
../../../../ext/sqlite3/database.c: In function ‘db_filename’:
../../../../ext/sqlite3/database.c:837: warning: implicit declaration of function ‘sqlite3_db_filename’
../../../../ext/sqlite3/database.c:837: warning: assignment makes pointer from integer without a cast
compiling ../../../../ext/sqlite3/sqlite3.c
../../../../ext/sqlite3/sqlite3.c: In function ‘init_sqlite3_constants’:
../../../../ext/sqlite3/sqlite3.c:96: error: ‘SQLITE_OPEN_URI’ undeclared (first use in this function)
../../../../ext/sqlite3/sqlite3.c:96: error: (Each undeclared identifier is reported only once
../../../../ext/sqlite3/sqlite3.c:96: error: for each function it appears in.)
../../../../ext/sqlite3/sqlite3.c:97: error: ‘SQLITE_OPEN_MEMORY’ undeclared (first use in this function)
gmake[1]: *** [sqlite3.o] Error 1

SQLITE_OPEN_URI and SQLITE_OPEN_MEMORY are not in /usr/include/sqlite3.h on SLES11SP4 (sqlite3-3.7.6.3-1.4.4.1).

I'll fix up a new pull request to handle this issue. I'm not sure if the other flags will have issues with older versions of SQLite3.

**Comment by tenderlove** [comment]
@jimflood I merged your PR.  Is this still a problem for you?

**Comment by jimflood** [comment]
I'm no longer on that project, so I have not been following the issue. My changes worked fine so I expect the merged code will be fine. I no longer have access to a SUSE VM.

**Comment by tenderlove** [comment]
@jimflood hah, ok.  I'll close this issue then.  Thank you!
